templates/settings.php

```diff
@@ -26,8 +26,8 @@
 						<td>
 							<p>
 								<label>
-									<input type="checkbox" name="activitypub_enable_users" id="activitypub_enable_users" value="1" <?php echo \checked( '1', \get_option( 'activitypub_enable_users', '1' ) ); ?> />
-									<?php \esc_html_e( 'Enable Author-Profiles', 'activitypub' ); ?>
+									<input type="radio" name="activitypub_actor_mode" id="activitypub_actor_mode" value="1" <?php echo \checked( '1', \get_option( 'activitypub_actor_mode', '1' ) ); ?> />
+									<strong><?php \esc_html_e( 'Only Author-Profiles', 'activitypub' ); ?></strong>
```

Comment:
No dashes in English necessary here, also putting Only at the end sounds a bit better.
```suggestion
									<strong><?php \esc_html_e( 'Author Profiles Only', 'activitypub' ); ?></strong>
```